tests/smoke/base.sh

```diff
@@ -49,17 +49,21 @@ elif command -v meson &> /dev/null; then
   pip install -e .
 elif command -v pdm &> /dev/null; then
   pdm install
+elif command -v hatch &> /dev/null; then
+  COMMAND_PREFIX="hatch run"
+  
 else
     # use setuptools
   pip install --editable .
 fi
 
-ipython kernel install --name "python3" --user
+$COMMAND_PREFIX ipython kernel install --name "python3" --user
+
+$COMMAND_PREFIX pre-commit install
 
-pre-commit install
-pre-commit run --all-files --verbose
+$COMMAND_PREFIX pre-commit run --all-files --verbose
 
-make docs-build
+$COMMAND_PREFIX make docs-build
```

Comment:
Were the tests failing when not using "COMMAND_PREFIX"?

Reply:
oh sorry .. maybe I miss understood your question .. when not using command prefix (empty variable) for the other backend works fine .. but for hatch .. it requests `hatch run`